.github/workflows/unit_tests.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   unit_tests:
     strategy:
       matrix:
-        python-version: [ 3.8, 3.9]
+        python-version: [ '3.9', '3.10', '3.11' ]
```

Comment:
may we also add `3.12` support?

Reply:
Probably. I think we just need to manually install `setuptools`, but that is an easy change too

Reply:
Looks like we have some incompatible dependencies. Should be fine for now since we're using 3.10 for all of our chatbot container builds

https://github.com/NeonGeckoCom/chatbot-core/actions/runs/14759289944/job/41435663976?pr=201